tutorials/getting-started/schema-guide.md

```diff
@@ -485,7 +485,9 @@ The default "local" time format, displayed to the user when editing, is American
 
 An `array` field has its own schema, and allows the user to create one or more objects that have the fields in that schema. These objects are stored as an array.
 
-This is useful for collections that clearly belong to a parent object, such as multiple homes for a person, and do not have any other relationships to other objects. If the objects in the array are also related to other types of objects, you should be using joins instead.
+This is useful for collections that clearly belong to a parent object, such as multiple homes for a person or tabs in a widget, and do not have any other relationships to other objects. If the objects in the array are also related to other types of objects, you should be using joins instead.
+
+The `limit` property on array fields will limit the number of items an editor can add to the array. So if you set `limit: 3`, they can only one to three items in that array.
```

Comment:
Grammar issue "they can only one to three items"

How about "they can only add up to three items"

Reply:
👨‍🏫